sdk/tables/Azure.Data.Tables/tests/samples/Sample1_CreateDeleteTable.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using Azure.Data.Tables.Tests;
+
+namespace Azure.Data.Tables.Samples
+{
+    [LiveOnly]
+    public partial class TablesSamples : TablesTestEnvironment
+    {
+        [Test]
+        public void CreateTable()
+        {
+            string storageUri = StorageUri;
+            string accountName = AccountName;
+            string storageAccountKey = PrimaryStorageAccountKey;
+            string tableName = "OfficeSupplies";
+
+            #region Snippet:TablesSample1CreateClient
+            var serviceClient = new TableServiceClient(
+                new Uri(storageUri),
+                new TableSharedKeyCredential(accountName, storageAccountKey));
+            #endregion
+
+            try
+            {
+                #region Snippet:TablesSample1CreateTable
+                serviceClient.CreateTable(tableName);
+                var client = serviceClient.GetTableClient(tableName);
```

Comment:
is there something we could show in Console here for the user? i.e. the name of the table or some properties?